.travis.yml

```diff
@@ -218,11 +217,15 @@ script:
       CMAKE_LINKER_FLAGS="-DCMAKE_LINKER_FLAGS=-fsanitize=${USE_SANITIZER}";
       CMAKE_C_FLAGS="-DCMAKE_C_FLAGS=-fsanitize=${USE_SANITIZER}";
     fi
-  - cd ..
   - |
-    if [ ${BUILD_SCHEMA} ]; then
-      git diff --exit-code; # check that no files (e.g. generated schemas) are changed
+    if [ "${SSL}" = "YES" ] && [ "${SECURITY}" = "YES" ]; then
```

Comment:
Methinks this is wrong. Besides, I think we should just remove `BUILD_SCHEMA` altogether. It takes no time, so why make it optional?

Reply:
The schema is different if SSL and/or security is not included so the check would fail. I'm afraid there's no way around that...

Reply:
Why did I not think of that!